spec/support/vcr_cassettes/lighthouse/direct_deposit/update/200_valid.yml

```diff
@@ -62,6 +62,6 @@ http_interactions:
     body:
       encoding: ASCII-8BIT
       string: '{"paymentAccount":{"accountType":"CHECKING","accountNumber":"1234567890","financialInstitutionRoutingNumber":"031000503","financialInstitutionName":"WELLS
-        FARGO BANK"},"controlInformation":{"canUpdateDirectDeposit":true,"isCorpAvailable":true,"isCorpRecFound":true,"hasNoBdnPayments":true,"hasIndentity":true,"hasIndex":true,"isCompetent":true,"hasMailingAddress":true,"hasNoFiduciaryAssigned":true,"isNotDeceased":true,"hasPaymentAddress":true}}'
+        FARGO BANK"},"controlInformation":{"canUpdateDirectDeposit":true,"hasCpClaim":true,"hasCpAward":true,"isCorpAvailable":true,"isCorpRecFound":true,"hasNoBdnPayments":true,"hasIndentity":true,"hasIndex":true,"isCompetent":true,"hasMailingAddress":true,"hasNoFiduciaryAssigned":true,"isNotDeceased":true,"hasPaymentAddress":true,"isEduClaimAvailable":true}}'
```

Comment:
The new standard is to hide the uri domain in vcr. The uri can be defined in the[ vcr configuration](https://github.com/department-of-veterans-affairs/vets-api/blob/16239c47337635c36aa1b7d2cfd474d469513227/spec/support/vcr.rb#L9). 

Ex:
http_interactions:
- request:
    method: put
    uri: <EVSS_BASE_URL>/services/direct-deposit-management/v1/direct-deposit?icn=1012666073V986297

Reply:
@RachalCassity Do you want me to update all of the profile direct deposit cassettes or just the ones changed in this PR?

Reply:
@RachalCassity I updated the cassettes that were modified in this PR and will continue to update cassettes my team owns in another PR.

Reply:
@RachalCassity Who should I list as owners for `config/initializers/integration_recorder.rb`?